src/views/developer-landing/styles.js

```diff
@@ -0,0 +1,135 @@
+import styled from "styled-components";
+import { theme } from "../../globalStyles";
+
+export const Wrapper = styled.section`
+  background: ${({ theme }) => theme.colors.mainBg};
+  padding: 0 var(--main-padding-x);
+  height: 100vh;
+  top: -100vh;
+  padding: 10rem 20rem 0rem 20rem;
+  display: flex;
+  flex-direction: column;
```

Comment:
We want it so that the user doesn't have to scroll when they're on their desktop. The `height: 100vh` is a good step at accomplishing that, but the padding makes it so that the height of the container exceeds the height of the browser window height. Decrease the padding-top, and use the `var(--main-padding-x)` for the horizontal padding. You have two lines for the padding already so just use one.

Even with a small padding-top, it will make the container exceed it. Add `box-sizing: border-box` so that the [additional padding is still within the 100vh height](https://www.w3schools.com/css/css3_box-sizing.asp).